src/logic/operate.js

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line import/extensions
+import Big from './big.mjs';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result;
+  switch (operation) {
+    case '+':
+      result = Big(numberOne).plus(Big(numberTwo));
+      break;
+    case '-':
+      result = Big(numberOne).minus(Big(numberTwo));
+      break;
+    case 'X':
+      result = Big(numberOne).times(Big(numberTwo));
+      break;
+    case '÷':
+      result = Big(numberOne).div(Big(numberTwo));
+      break;
+    default:
+      result = Big(numberOne).times(Big(numberTwo).times(0.01));
```

Comment:
[Optional]
I think you can create two variables after line no.5 to convert the numbers. And then you can directly use those variables. It will clean the code bit more.
```js
const one = Big(numberOne);
const two = Big(numberTwo);
// then place variables where they are needed.
```